src/VisualStudio/Core/Impl/ProjectSystem/CPS/CPSCodeModelFactory.cs

```diff
@@ -0,0 +1,21 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System.ComponentModel.Composition;
+using Microsoft.VisualStudio.LanguageServices.ProjectSystem;
+
+namespace Microsoft.VisualStudio.LanguageServices.Implementation.ProjectSystem.CPS
```

Comment:
All types in this project are under the implementation namespace - CPS at the end is to match the folder name under which the files are placed. We should never have to access the implementing types directly.